tutorials/web-demo/webpack.config.js

```diff
@@ -28,6 +28,12 @@ export default {
 				loader: 'sass-loader'
 				}
 			]
+		},
+		{
+			test: /\.(wasm)$/,
+			generator: {
+				filename: '[name].wasm?v=[hash]'
```

Comment:
Not sure if this works on windows? The `?v=[hash]` gets dropped on creation.

As long as this works on ubuntu for the purposes of the CI/uploading to GCP to avoid upsetting the cache, then I have no issue with it.

Otherwise, I think the main change will have to be in the JS for loading the .wasm file, and updating that with every build to use a `.wasm?[hash]` query for `fetch`.

Reply:
Wait, I just realized that this DOES update the script. Okay, I approve

Reply:
Yes, it generates files without the query string in the file name, but it loads them with the query string at runtime, according to the web inspector.